### PR TITLE
[ADD] new module 'user_all_groups' to add automatically some users to all groups

### PR DESCRIPTION
**Pull request by legalsylvain**
supersed #538 

@bealdav : https://github.com/OCA/server-ux/pull/538#discussion_r1162784210 fixed.
@fcayre : https://github.com/OCA/server-ux/pull/538#issuecomment-1503260508. I added a section for the exclusive group.

could you take a look ? thanks ! 

CC : @quentinDupont

### CODE REVIEW

File: user_all_groups/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  initial configuration.
+
+.. figure:: ../static/description/view_res_users_form.png
+
+
+**Note**
+
+- In Odoo Core, some groups are exclusive. This is the case for the three base groups
+  "Access Rights / Portal / Public" and the two account groups "Tax display B2B / Tax display B2C".
+  All that groups are not impacted by this module.
```
Comment (fcayre) [inline user_all_groups/readme/DESCRIPTION.rst]: This formulation is unclear (to me at least), I would reword with something like:

> Users marked as "Member of all Groups" are NOT added to these exclusive groups.

**Review by fcayre — APPROVED** [review]
I added a non-blocking suggestion regarding the documentation.

File: user_all_groups/readme/DESCRIPTION.rst (hunk 1 above)
Comment (legalsylvain) [reply user_all_groups/readme/DESCRIPTION.rst]: done. Thanks.

File: user_all_groups/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+* Sylvain LE GAL <https://twitter.com/legalsylvain>
```
Comment (bealdav) [inline user_all_groups/readme/CONTRIBUTORS.rst:1]: s / twitter / mastodon  ;-)

**Review by bealdav — APPROVED** [review]
Nice job, really thanks

**Review by bguillot — APPROVED** [review]
Thanks
